.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+name: Release image
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    name: Build and push image
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Bump version and push tag
+      id: tag_version
+      uses: mathieudutour/github-tag-action@v5.6
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Create a GitHub release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ steps.tag_version.outputs.new_tag }}
+        release_name: Release ${{ steps.tag_version.outputs.new_tag }}
+        body: ${{ steps.tag_version.outputs.changelog }}
+
+    - name: Build Image
+      id: build-image
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: operator-pipelines-images
+        tags: latest ${{ github.sha }} ${{ steps.tag_version.outputs.new_tag }}
```

Comment:
I added a both commit sha and version name so us or users can use any of these. Since it is just a tag it doesn't store a duplicated layers in registry.

Reply:
Thanks, great :+1: